src/PIL/TiffImagePlugin.py

```diff
@@ -1128,6 +1131,20 @@ def getxmp(self):
         """
         return self._getxmp(self.tag_v2[700]) if 700 in self.tag_v2 else {}
 
+    def getphotoshopblocks(self):
```

Comment:
Naming: I see this follows the other one here -- `getxmp` -- but how about `get_photoshop_blocks` for this new one?

Docs: let's at least add a docstring and release notes. Would any other documentation be useful?

(Docstring goes here:)

![image](https://user-images.githubusercontent.com/1324225/153193696-7eb5a679-913d-4c67-95ff-8c8801359f8e.png)

https://pillow--6030.org.readthedocs.build/en/6030/reference/plugins.html#PIL.TiffImagePlugin.TiffImageFile.getphotoshopblocks